src/queries/index.js

```diff
@@ -76,6 +77,8 @@ export const getQuery = (query, filterOptions = {}) => {
     [QUERY_TYPES.PUBLIC_JSON_FILE]: GET_PUBLIC_JSON_FILE,
     [QUERY_TYPES.WASTE_ADDRESSES]: WASTE_ADDRESSES,
     [QUERY_TYPES.WASTE_STREET]: WASTE_STREET,
+    [QUERY_TYPES.VOUCHERS]: GET_VOUCHERS,
+    [QUERY_TYPES.VOUCHER]: GET_VOUCHER,
```

Comment:
we always have singular first, as it also should be the alphabetical order

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/510bdb9e7e17a14badeca2b70989a4ef191a7e92